src/Core/Components/DataGrid/Columns/SelectColumn.cs

```diff
@@ -179,6 +179,15 @@ public DataGridSelectMode SelectMode
     [Parameter]
     public override GridSort<TGridItem>? SortBy { get; set; }
 
+    /// <summary>
+    /// Allows to clear the selection.
+    /// </summary>
+    public void ClearSelection()
```

Comment:
Could you create a `ClearSelectionAsync` method, returning `Task.CompletedTask` for now. But better for the future.

Reply:
Done